Reference/Searching/Examine/Indexing/index.md

```diff
@@ -323,16 +322,17 @@ using Umbraco.Cms.Core.Composing;
 using Umbraco.Cms.Core.DependencyInjection;
 using Umbraco.Cms.Infrastructure.Examine;
 
-namespace MySite.MyCustomIndex
+namespace Umbraco.Docs.Samples.Web.CustomIndexing
 {
     public class ExamineComposer : IComposer
     {
         public void Compose(IUmbracoBuilder builder)
         {
             builder.Services.AddExamineLuceneIndex<ProductIndex, ConfigurationEnabledDirectoryFactory>("ProductIndex");
-            builder.Services.ConfigureOptions<ConfigureCustomIndexOptions>();
```

Comment:
Should this be deleted? 🤔 
There's still a section at the top of the article called "Creating a ConfigureOptions class".

Reply:
In this tutorial, we're not actually adding any `ConfigureCustomIndexOptions` so there is no need to add that line. And on line 46 we're already explaining if we add 'ConfigureNamedOptions' then the next code block shows how to register it :)

Reply:
Oh, alright 🤔 
Then I wonder why it was added in the first place!

Reply:
It was I who added that while trying the tutorial then I realized we don't need that so I removed :)